### PR TITLE
[Renovate] Schedule renovate update every Monday

### PR DESCRIPTION
**Pull request by KyleJu**
Switch renovate to batch updates every Monday. Currently Renovate sends updates whenever there is one available.

See the [schedule](https://docs.renovatebot.com/presets-schedule/#scheduleearlymondays) syntax

### CODE REVIEW

**Review by stephenmcgruer — APPROVED** [review]
Can I clarify; when I read "on demand" I usually interpret it as "someone has to make a request for this to happen". However here I think by "on demand" you mean "renovate will schedule an update instantly whenever there is one".

If that's correct, I would suggest rewording the PR description a little to clarify that we are switching to batch updates (which I believe is fairly common terminology), rather than as-available updates.

**Comment by KyleJu** [comment]
> Can I clarify; when I read "on demand" I usually interpret it as "someone has to make a request for this to happen". However here I think by "on demand" you mean "renovate will schedule an update instantly whenever there is one".
> 
> If that's correct, I would suggest rewording the PR description a little to clarify that we are switching to batch updates (which I believe is fairly common terminology), rather than as-available updates.

Updated the description!
